pilot/pkg/proxy/envoy/v2/eds_test.go

```diff
@@ -269,6 +269,12 @@ func TestDeleteService(t *testing.T) {
 	}
 }
 
+func fullPush(server *bootstrap.Server) {
+	server.EnvoyXdsServer.Push(&model.PushRequest{Full: true})
+	// TODO: See if we can disable debounce completely for tests to avoid sleep.
+	time.Sleep(200 * time.Millisecond) // Wait till push debounce is stable.
```

Comment:
We should disable debounce for ut instead of sleeping here

Reply:
I could not find an easy way to disable with the current code as it is set in `init()` func in discovery.go. I tried to overwrite it after it is set but ran in to data race issues. So that is why I added that TODO to disable. I can do it in a separate PR.

If you have any idea to disable easily with current code, Let me know. I will change it.